### ISSUE TITLE
test_mask_reducer failing on circle CI

### ISSUE THREAD

**Issue by jeromedockes** [nilearn/nilearn#2139]
testing the mask reducer from nilearn.decompositions.base sometimes fails on circleCI, e.g.
https://travis-ci.org/nilearn/nilearn/builds/583674961?utm_source=github_status&utm_medium=notification

the process gets a SIGABRT

could be due to the recent unvendoring of joblib, although this error started more recently

so far I haven't been able to reproduce this locally

**Comment by jeromedockes** [comment]
actually it seems to be due to an issue with intel openmp 2019.5:

https://github.com/ContinuumIO/anaconda-issues/issues/11294

https://github.com/scikit-learn/scikit-learn/pull/15020/files

and here is the fix used by scikit-learn:

https://github.com/scikit-learn/scikit-learn/blob/6bd8df099f37205864eab1759f77b6070577f6e7/sklearn/__init__.py#L63
